src/client/components/UserBar/UserBar.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { ENDPOINT_URL } from '../../common/variables';
+import { logoutCurrentUser } from './actions';
+import { getCurrentUser } from '../../selectors';
+import { UserPropType } from '../../common/propTypes';
+
+class UserBar extends Component {
+  handleLogOut = () => {
+    const { logoutCurrentUser } = this.props;
+
+    /**
+     *  Wait 300ms for backend service to
+     *  logout, than execute logout on front.
+     */
+    setTimeout(() => logoutCurrentUser(), 300);
+  };
+
+  render() {
+    const {
+      currentUser: { avatar, name }
+    } = this.props;
+    return (
+      <div className="user-bar">
+        <div className="user-bar__wrapper">
+          <img alt="User avatar" className="user-bar__avatar" src={avatar} />
+          <div className="user-bar__logged">
+            <span className="user-bar__user-data">Logged as:</span>
+            <span className="user-bar__user-data">{name}</span>
+          </div>
+          <a
+            className="user-bar__logout"
+            href={`${ENDPOINT_URL}/logout`}
```

Comment:
Please don't do the logout actions on the GET request. It is possible to prepare the following request and place it anywhere (e.g. in an email).
```
<img src="http://eoc/logut" />
```

Reply:
When you will do the logout on the POST request, please wait untill it finishes and do the redirect to home page (please ensure, that the page will be reloaded - it should clear the redux store, and all temporary data)